src/ReachSets/ContinuousPost/TMJets/post.jl

```diff
@@ -47,18 +109,18 @@ function post(𝒜::TMJets,
 
     info("Reachable States Computation...")
     @timing begin
-        tTM, xTM = validated_integ(f!, q0, δq0, t0, T, orderQ, orderT, abs_tol,
-                                   maxsteps=max_steps, check_property=property)
+        tTM, xTM, vTM = validated_integ(f!, q0, δq0, t0, T, orderQ, orderT, abs_tol,
+                                        maxsteps=max_steps, check_property=property)
     end
 
     # convert to hyperrectangle and wrap around the reach solution
-    N = length(xTM)
-    SET_TYPE = Hyperrectangle{Float64, SVector{n, Float64}, SVector{n, Float64}}
-    Rsets = Vector{ReachSet{SET_TYPE}}(undef, N-1)
-    @inbounds for i in 1:N-1
-        Hi = convert(Hyperrectangle, xTM[i])
-        t0 = tTM[i]; t1 = tTM[i+1]
-        Rsets[i] = ReachSet(Hi, t0, t1)
+    output_type = 𝑂[:output_type]
+    if output_type == Hyperrectangle
+        Rsets = _to_hyperrectangle(tTM, xTM, n)
+    elseif output_type == IntervalBox
+        Rsets = _to_intervalbox(tTM, xTM, n)
+    elseif output_type == Zonotope
+        Rsets = _to_zonotope(tTM, vTM, n)
```

Comment:
The Julian way would be to pass the output type to the conversion method:
```julia
Rsets = _convert_flowpipe(tTM, xTM, n, 𝑂[:output_type])

...

function _convert_flowpipe(tTM, vTM, n, ::Type{Hyperrectangle})
    ...
end
function _convert_flowpipe(tTM, vTM, n, ::Type{IntervalBox})
    ...
end
function _convert_flowpipe(tTM, vTM, n, ::Type{Zonotope})
    ...
end
```

Reply:
Ok, but the "problem" with this approach is that the different implementations do not receive the same arguments (one of them receives xTM, the boxes, while the other receives vTM, the Taylor models).

Reply:
Fair enough, I did not see that.